menas/ui/service/SchemaService.js

```diff
@@ -92,25 +92,26 @@ var SchemaService = new function() {
 			uri += "/" + encodeURI(iVersion)
 		}
 		
-		Functions.ajax(uri , "GET", {}, function(oData) {
-			if(Array.isArray(oData)) {
-				var err = "Disabling schema failed. Clear the following dependencies first:\n";
-				for(var ind in oData) {
-					err += "\t - " + oData[ind].name + " (v. " + oData[ind].version + ")";
-				}
-				sap.m.MessageBox.error(err)
-			} else if(typeof(oData) === "object") {
-				sap.m.MessageToast.show("Schema disabled.");
-				if(window.location.hash != "#/schema") {
-					window.location.hash = "#/schema"
-				} else {
-					SchemaService.getSchemaList(true, false)
-				}
-			}
-		}, function() {
-			sap.m.MessageBox.error("Failed to disable schema. Ensure no mapping tables or datasets use this schema(and/or version)")
-		})	
-	};
+    Functions.ajax(uri , "GET", {}, function(oData) {
+      sap.m.MessageToast.show("Schema disabled.");
```

Comment:
Is it just badly showing or is the indentation moved too far to the left?

Reply:
I think a `return` is bigger antipattern in _Scala_ then a long `if` with short `else`.

Reply:
Return is not and antipatern and scala does emphasize on readability, which this provides.

Reply:
I think this conversation is taking place on the wrong thread. 
As to the original comment, the indentation is 2spaces (showing 2 spaces), the stuff around it is tabbed (showing 4-spaces). This is just the way gtihub is showing it. I didn't want to edit it beyond what was necessary because there's big changes in #79 and I didn't want to deal with the conflicts.